pkgs/racket-doc/scribblings/reference/match.scrbl

```diff
@@ -107,6 +108,8 @@ In more detail, patterns match as follows:
        (match '(1 (x y z) 1)
          [(list a b a) (list a b)]
          [(list a b c) (list c b a)])
+       (match #false
```

Comment:
Is there a reason for using `#false` here rather than `#f`? It seems unusual enough to be confusing to a casual Racketeer.

Reply:
I like `#false`, that's all.

Reply:
I think I prefer it to `#f`, too, in a vacuum, so I sympathize… but I think I prefer consistency more. But I also won’t die on this hill. :)